codespell_lib/data/dictionary.txt

```diff
@@ -3145,20 +3146,26 @@ dashs->dashes
 databae->database
 databas->database
 databse->database
+datastrem->datastream
 datea->date, data,
 daty->data, date,
 daugher->daughter
+dchp->dhcp
```

Comment:
Can you capitalise the RHS please, the system will lower case it for now, but it future proofs things.